reya_trading/resources/orders.py

```diff
@@ -0,0 +1,309 @@
+        order_request = TriggerOrderRequest(
+            account_id=account_id,
+            market_id=market_id,
+            trigger_price=trigger_price_val,
+            price=price_val,
+            is_buy=is_buy,
+            trigger_type=trigger_type,
+            size="",  # SL/TP orders typically have 0 size
+            reduce_only=False,
+            nonce=nonce,
+            signature=signature,
+            signer_wallet=self.signature_generator._public_address
+        )
+        
+        # Make the API request
+        endpoint = "api/trading/create-order"
+        response_data = self._post(endpoint, order_request.to_dict())
+        
+        return OrderResponse.from_api_response(response_data)
+    
+    def cancel_order(self, order_id: str) -> OrderResponse:
+        """
+        Cancel an existing order.
+        
+        Args:
+            order_id: ID of the order to cancel
+            
+        Returns:
+            API response for the order cancellation
+            
+        Raises:
+            ValueError: If the API returns an error
+        """
+        if self.signature_generator is None:
+            raise ValueError("Private key is required for cancelling orders")
+        
+        # Sign the cancellation request
+        signature = self.signature_generator.sign_cancel_order(order_id)
+        
+        # Create the cancellation request
+        cancel_request = CancelOrderRequest(
+            order_id=order_id,
+            signature=signature
+        )
+        
+        # Make the API request
+        endpoint = "api/trading/cancel-order"
+        response_data = self._post(endpoint, cancel_request.to_dict())
+        
+        return OrderResponse.from_api_response(response_data)
+    
+    def get_orders(
+        self,
+        wallet_address: str,
+    ) -> Dict[str, Any]:
+        """
+        Get filled orders for a wallet address.
+        
+        Args:
+            wallet_address: The wallet address to get orders for
+            
+        Returns:
+            Dictionary containing orders data and metadata
+            
+        Raises:
+            ValueError: If the API returns an error
+        """
+
+        # Make the API request
+        endpoint = f"api/trading/wallet/{wallet_address}/orders"
+        response_data = self._get(endpoint)
+        
+        return response_data
+        
+    def get_conditional_orders(
+        self,
+        wallet_address: str,
+    ) -> List[Dict[str, Any]]:
+        """
+        Get conditional orders (limit, stop loss, take profit) for a wallet address.
+        
+        Args:
+            wallet_address: The wallet address to get orders for
+            
+        Returns:
+            List of conditional orders
+            
+        Raises:
+            ValueError: If the API returns an error
+        """
+        
+        # Make the API request
+        endpoint = f"api/trading/wallet/{wallet_address}/conditionalOrders"
```

Comment:
hmm, think we should rename this one to /orders (once orders is renamed to /trades)
and within that we can surface all order updates 

smth similar to how it's done in binance: https://developers.binance.com/docs/derivatives/usds-margined-futures/user-data-streams/Event-Order-Update ?